htp/htp_request_generic.c

```diff
@@ -93,8 +93,8 @@ htp_status_t htp_process_request_header_generic(htp_connp_t *connp, unsigned cha
             // Don't use string comparison here because we want to
             // ignore small formatting differences.
 
-            int64_t existing_cl = htp_parse_content_length(h_existing->value);
-            int64_t new_cl = htp_parse_content_length(h->value);
+            int64_t existing_cl = htp_parse_content_length(h_existing->value, NULL);
```

Comment:
this commit looks like it does something else than the commit message says

Reply:
Indeed, with the latest merges, this one should now be squashed with 
282deb0